### DISCUSSION TITLE
Instant upload vs Schedule Sync

### DISCUSSION THREAD

**Discussion by ztevo** [trakt/showly#724] [Q&A] [answered]
Hi. Could someone kindly explain how instant upload works and why it is advised (via a popup in the app) not to use alongside scheduled sync?

I will usually make updates via Showly app, but there are times when I update Trakt directly on the website. For that reason, I want to be sure a two way sync is taking place - does that mean I should disable Instant Upload?

Thank you.

**Comment by michaldrabik** [comment, marked as answer]
Basically if you have scheduled sync set up it will sync both ways every X hours. There is no need to have Instant Upload active then since everything will be synced anyway - just a bit later.

You can still use both options - warning is just an information.

**Reply by ztevo** [reply]
Great thanks that's what I expected/hoped. Stevo.
